index.js

```diff
@@ -513,10 +498,16 @@ module.exports = function (log, indexesPath) {
     done((err, recs) => {
       // FIXME: handle error better, this cb() should support 2 args
       if (err) return console.error(err)
+      const seek = op.data.seek
```

Comment:
This is tiny optimization somewhat unrelated to this PR, but good to have. Since the for-loop below is a hot path, doing many times `.` into object fields is wasteful, so we hold a reference to the seek function once.